emulator/blockchain.go

```diff
@@ -692,6 +694,17 @@ func configureBootstrapProcedure(conf config, flowAccountKey flowgo.AccountPubli
 	options = append(options,
 		fvm.WithInitialTokenSupply(supply),
 		fvm.WithRestrictedAccountCreationEnabled(false),
+		fvm.WithTransactionFee(fvm.DefaultTransactionFees),
```

Comment:
@m-Peter Could you please bring back the comment describing where these magic values come from, i.e. add
```
		// This enables variable transaction fees AND execution effort metering
		// as described in Variable Transaction Fees: Execution Effort FLIP: https://github.com/onflow/flow/pull/753)
```

Reply:
@turbolent Sure thing :innocent: